tangelo/toolboxes/optimizers/rotosolve.py

```diff
@@ -15,10 +15,36 @@
 import numpy as np
 
 
-def rotosolve_step(func, var_params, i, *func_args):
+def extrapolate_expval(theta, m_0, m_minus, m_plus, phi=0.0):
+    """Extrapolates the expectation value of an observable
+    M with respect to a single parameterized rotation (i.e.
+    RX, RY, RZ) with angle theta. The extrapolation is
+    computed based on samples taken at the angles
+    phi, phi+pi/2, and phi-pi/2. This function used the
```

Comment:
uses

Reply:
In commit [88c127f](https://github.com/goodchemistryco/Tangelo/pull/392/commits/88c127fe8a997d1f1d510892dcd40c9f94b4b4af) I have changed the docstring to:

*The extrapolation uses samples taken at the angles phi, phi+pi/2, and phi-pi/2.*